fuse/data/ops/tests/test_op_visprobe.py

```diff
@@ -1,284 +0,0 @@
-import unittest
```

Comment:
So where is the test that runs the notebook?

Reply:
i'l add it